slacktee.sh

```diff
@@ -674,8 +681,8 @@ function check_configuration()
 		err_exit 1 "curl is not installed. Please install it first."
 	fi
 
-	if [[ $webhook_url == "" ]]; then
-		err_exit 1 "Please setup the webhook url of this incoming webhook integration."
+	if [[ $webhook_url == "" && $token == "" ]]; then
+		err_exit 1 "Please setup the authentication token or the incoming webhook url."
```

Comment:
I'd add (deprecated) to the webhook url portion here

Reply:
Great suggestion! I will add it

Reply:
Added!